common/src/main/java/org/opensearch/ml/common/MLModel.java

```diff
@@ -89,6 +93,7 @@ public class MLModel implements ToXContentObject {
     public static final String CONNECTOR_FIELD = "connector";
     public static final String CONNECTOR_ID_FIELD = "connector_id";
     public static final String GUARDRAILS_FIELD = "guardrails";
+    public static final String MODEL_INTERFACE_FIELD = "model_interface";
```

Comment:
```suggestion
    public static final String MODEL_INTERFACE_FIELD = "interface";
```

Reply:
changed

Reply:
interface is a general/high level word which can be misleading, a suggestion would be `api_specs` which can clearly imply this field is defining `specifications`.